os/gres/testdata/data/data.go

```diff
@@ -3,7 +3,7 @@ package data
HTQcs3aoMmrNTVfOHCfwCzelV2cq3v2gh7iU98op3tVqwjlkyw3XVoH96a+x+3XSNtVrIc6OFWiyv9zTSaR9kY4qz9g3bBVRr9Svubn0yQ4PA/aRlsDhd4XIv1c9CDnmbPNYw/GaN+NEbM8+U3xW/tiqFp0jpqe9Ay6u/8t/pwE+kydVNJzflyBbs0lMpcSz713wqqIgP+NxoWct6+qLBV7exvW6Nr5qsv1S7uYZdC3hoMfZpFO+Sh9UJjI2ywXSbxPOH67p2dJVe13zYioDsq/J9MexCy8tkYjVwTbvS+tOcBWs3fbxo+ANwoCtuxfBRebuqxEFp2wlw0R8VOTHwrHyoHrTX+rCoZNpxxkErpjy+vKin6e9GFEr6m1uZShqSuWOLt4lMZnGzk5jwcGLJ37X/vpGnv8on5vpCf3Ht47GJvtwJTzIzRluKB5scEBdYxCMdl/fIuspgc9nun5C+jVOWmaHcqNl+g/tnAAD1gO3NuxjI6idZGivZ13zqTN191e9ySfJZ/SjMqOxtqM8HB1jgx4xB55m2d4P3d/HmG8WbnL0XaxF5yli7RfRzPOEQYNMbXyOblZ4kJJKx5nn9dWnfjiO7A1v18AIeAlGM1efvmFu0ycdiYslSepUrQ8PyNTlKOgMk8shNkWEs3AqK3IVMdRrtmFwpFv1SYXXe9+v3zxgWGt6KTGHEx9qev0Aekvt5siO3MHbO0SsP6/fjnormn9iEGEQO1zIez9Dy5HBOpDm+H0zgyOk8rxAuYk3n7s/n31Hd6WYa0h4D/ZssC2mgId+B0u3XO8QOnKNZ64YZ8OjYefH6cyXPnhgMCe6bg7HNBSjj9fX5HSDePduDVXW8q32dH5vcLL6BzWfymK8keVbVM+9agq5whqp5snhHqbiljp8IKAVbaMV16iKYXS5kbRhl2qSQEH2+HaiT5pd3x7WyD7bOASm+uzu/tw+lWg9NmTNnbE9zye5fw+Z+24HZLABcJl9oTcfjt9vPnh7nI/UIQLkDVWyqsa1WIiN7pMjOrgIDNCcEc9XW7s+g2Xjc4tol4MXJw1j+hINtYr2RoQnPN8XFu5p1jYeWLjVJycpi8+BZMQgWNiX8uPut7SuwpvhpxQK2D8KIk4Lhhb4+7XrD67T2cuwwb+/9hHR0r/3m5eDILPjs7x0YZZ4m+CdVS8V1icolJ7yutH1bPeP3gnW2YqwPT/MzgEAHi36yWiqomXMXFYDZkDAuR5xwRJw06/tFEwZlDRGWYiVaZuXEjE6qUZi/350NdeuIOo00P6eZz2NEQMiAIANC7648pATumB93bz+/dz4Z6+wIlSgUHZueALWCY/zgNeRkmago19BxJgb6VZVb9EhEjFIk5TUtqwy6WzP4RFW96EhT/asmmrp7M8paQaVmHupc7O6zYzv+hQAADILChKiJsjBzY2wgJqKapTOnBTSCIc7NR1mxUrai4+aqOo4hMPaU9dRrq2vN6NjZkwonU064kWgqmXM/0LdVgAAanlaZv4P15JlTBjwXhnKdsS8VKysMvNzVqDxvVDxo9YsLCws8puvbla8d7R3FK8Z+vc9FpXQ6NE7m+XHOI0zwrq6ourfB11/GWVGv/lCQb3wnbd7pdYdD1kjrzMqHm2UYIKu4InyXBUe7hodRSLprpIlmYRflmbfWdrwepdcV5J9DPbaxZhbB4MaQDJdCn7d8RCMy1DhCbrfhZZXXdwYBQAYW05jo5fR2FQ8Q1PsI7Kembsf97oMVfKSfR38Ay5KvZIF74k5Lknlph3/AxelXkhJqyJiqsT1dWYbs7ZCrKzS2OTV/dS2XaWdbaIO7cE8X24zb7nT1hErdHjuMzvdaAfjVF8aLLhca6hpccU6LefL+aaF8KZ/yFDw0mtoSDYbbiVRv2f6O/rK/5EOTYWObOVsVX3byOnmnzBsZHQLnDQraHho5zRDM1hTls/+Swyc+rl4OowcDxqZ4oHh3Z2HR5bjmoOinLGaHdP/2qm1AlBNXM3J+Qv9wAwabeKDyQmfw6CWuCIHgyaRuGBgUjRggWzUQoWxwAojwnAoVEV5Sj4L0LOTnRZQSjHB1wiaNoKboge5nEqKiRwMGjNigYFlzILB40oLuUEHc2NqhyfLJi3VB9WZS9EL+gDNEMF9OAG5nEo2iRwMGh6C+9A6CwYPIS3dB3l6QJ44ou4DK8wHd3pAJXEEVw8NB8GteAhHoJQ4IgeDBoHgYDwMYLF00dJr84KCwZJEcDnQoA9cTi4cgVKSiBwMGumBg21iBIslhZZe21koGCwTBJcDje3wwuSUwREoZILIsaD5HDgWmgksEvtZqLKVsMqioVjk+R6yfQqSvYGfTW/ngVDK95DjQbM1cDxdZrCE/M7Si8wnw4Plc+CioGkYfpioX/NAKORzyOGg+RU43A0WsHigZqFNiQm2KW1nBZQiMHA10C/+8BPTd/ZyihEYchxo7ASO8wmGQ55xIceBZkk4YThWbIB6ZIUcBpoMgT/kEGEwFGIo5FDQ5AY3DEqTHSwYLFloqZhhS9UPRVrSWsG9kV4JqOc+yGGgYQs4TAwUhjzYMU8NJD8Bh1nJAajnNMhhoEEFuL1BUBgK0YulIw2QIZGnKpZ+T9lwAkqBiaU+9NziBJQCE/A6oPGFVbA6miGXUwhMLKSDBabDmgtQzj6QnV2QaSDc0RswgPnZB3Ik6BSOA4YUKwCozhKX/gDmJAjI53dwAdDR2mqYgERBsOj8biFneWDO8gqBxYZzcGHQuZgITJgNFShqw7l5L46QSZcQ/H6ghkxpOjLv/IIMreCwvhvB0odm5LDQEREcll4YLH3+tfSV6qUMi6a6UtBpEFyhoghY+rSJHBY6+IHD3qEGuxQ/oTMcOCzXJrD0GdJCfq6B+XmDGizZOAguEzqr2QST2bMQHqVxEDk0dC4Dh7beDP5s9LPQRsQG24gqyKAhGxI9w9QfqQAV0MgFwErRqd/+LwAA//+ohVRS+zcAAA=="); err != nil {
```

Comment:
do not update the testdata of package res.

Reply:
sorry, this should be a change caused by me accidentally triggering gofmt